package day5;

public class ExceptionTest {

	public static void main(String[] args) {
		// ArrayIndexOutOfBoundsException
		int i = Integer.parseInt(args[0]); // 
		int j = Integer.parseInt(args[1]); //이 값이 없어서 ArrayIndexOutOfBoundsException
		System.out.println(i/j);
		System.out.println("main 종료");

	}

}
